### ISSUE TITLE
Crash on Android 4 devices/emulators

### ISSUE THREAD

**Issue by keima** [yshrsmz/LicenseAdapter#50]
This lib's minSdkVersion is 15.
But example app crashed on Android 4.4 emulator (see Logcat).

My colleague suppose that old device/emulator couldn't treat `drawable-anydpi` directory. (ref: https://stackoverflow.com/a/47300666/2707149 )
LicenseAdapter uses its directory.

<details>
<summary>Logcat</summary>

```
08-02 16:42:20.061 3537-3537/net.yslibrary.licenseadapter.example E/AndroidRuntime: FATAL EXCEPTION: main
    Process: net.yslibrary.licenseadapter.example, PID: 3537
    android.view.InflateException: Binary XML file line #43: Error inflating class ImageView
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:714)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:756)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
        at net.yslibrary.licenseadapter.LicenseAdapter.onCreateViewHolder(LicenseAdapter.java:79)
        at net.yslibrary.licenseadapter.LicenseAdapter.onCreateViewHolder(LicenseAdapter.java:32)
        at android.support.v7.widget.RecyclerView$Adapter.createViewHolder(RecyclerView.java:6685)
        at android.support.v7.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:5869)
        at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5752)
        at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5748)
        at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2232)
        at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1559)
        at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1519)
        at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:614)
        at android.support.v7.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3812)
        at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:3529)
        at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:4082)
        at android.view.View.layout(View.java:14817)
        at android.view.ViewGroup.layout(ViewGroup.java:4631)
        at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1055)
        at android.view.View.layout(View.java:14817)
        at android.view.ViewGroup.layout(ViewGroup.java:4631)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
        at android.view.View.layout(View.java:14817)
        at android.view.ViewGroup.layout(ViewGroup.java:4631)
        at android.support.v7.widget.ActionBarOverlayLayout.onLayout(ActionBarOverlayLayout.java:443)
        at android.view.View.layout(View.java:14817)
        at android.view.ViewGroup.layout(ViewGroup.java:4631)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
        at android.view.View.layout(View.java:14817)
        at android.view.ViewGroup.layout(ViewGroup.java:4631)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1671)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1525)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1434)
        at android.view.View.layout(View.java:14817)
        at android.view.ViewGroup.layout(ViewGroup.java:4631)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:388)
        at android.view.View.layout(View.java:14817)
        at android.view.ViewGroup.layout(ViewGroup.java:4631)
        at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:1987)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1744)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1000)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5670)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:761)
        at android.view.Choreographer.doCallbacks(Choreographer.java:574)
        at android.view.Choreographer.doFrame(Choreographer.java:544)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:747)
        at android.os.Handler.handleCallback(Handler.java:733)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:136)
        at android.app.ActivityThread.main(ActivityThread.java:5017)
        at java.lang.reflect.Method.invokeNative(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:515)
    	at com.android.internal.os.ZygoteInit$MethodAndArgsCalle
```

</details>

**Comment by SUPERCILEX** [comment]
Do you have `vectorDrawables.useSupportLibrary = true` enabled?

**Comment by keima** [comment]
@SUPERCILEX 
Conclusion: No effect at all...

In example app of LicenseAdapter,
https://github.com/yshrsmz/LicenseAdapter/blob/master/example/build.gradle
`vectorDrawables.useSupportLibrary = true` is not set.
So I added it and run on Android 4.4 emulator but caught crashed.

**Comment by SUPERCILEX** [comment]
You're right, and this will be fixed in #51

**Comment by yshrsmz** [comment]
@keima v2.1.2 is released with the fix. let me know if you find any issue. thanks!
